examples/e2e-tests/kubernetes/release/default/definition.json

```diff
@@ -4,6 +4,7 @@
     "orchestratorProfile": {
       "orchestratorType": "Kubernetes",
       "kubernetesConfig": {
+        "useManagedIdentity": true,
```

Comment:
The cluster configuration we maintain to run PR E2E tests in fact has the required criteria to demonstrate the bug; we merely need to enable system-assigned identity (enabled above) to reproduce.